### ISSUE TITLE
Swallow user input when HookManager is active

### ISSUE THREAD

**Issue by TheoRet** [JeffHoogland/pyxhook#15]
Hello,

thanks for this nice module.

I want to write a script that swallows/catches the user input so that it is not sent to the current window, while the HookManager is active.
Can this be done with the current pyxhook version? (I don't understand much of the source code.)
If not, then this is a feature request for implementing it ;-)

ps: Sorry if this question does not belong here, but I dont know where else to ask.

**Comment by JeffHoogland** [comment]
I don't believe so. Feel free to submit patches, I don't have a need for a feature like this personally.

**Comment by Aareon** [comment]
Something of interest to the implementation of this feature can be found [here](https://github.com/PyUserInput/PyUserInput/blob/ac2d4c7a7f4b1a72e70b1a2ef8925d5312fb12bc/pykeyboard/x11.py#L246), in the PyUserInput repository.
